### ISSUE TITLE
[Bug]: dkwm wm-restack?

### ISSUE THREAD

**Issue by ACR-Jeff** [polybar/polybar#3129]
### Checklist

- [X] I have read the appropriate section in the [contributing guidelines](https://github.com/polybar/polybar/blob/master/CONTRIBUTING.md)
- [X] I believe this issue is a problem with polybar itself and not a misconfiguration on my part
- [X] I have searched for other open and closed [issues](https://github.com/polybar/polybar/issues?q=is%3Aissue) that may have already reported this problem
- [X] I have checked the [known issues](https://github.com/polybar/polybar/wiki/Known-Issues) page for this problem.
- [X] I have followed the [debugging guide](https://github.com/polybar/polybar/wiki/Debugging-your-Config) to narrow down the problem to a minimal config.

### Steps to reproduce

1. install dk window manager
2. launch polybar or restart with polybar in autostart config
3. launch a window

### Minimal config

```dosini
[global/wm]
margin-top = 0
margin-bottom = 0

[settings]
screenchange-reload = true

[colors]
format-foreground = #bfbfbf
format-background = #2e2e2e

background = #2e2e2e
foreground = #fefefe
black = #000000
white = #fefefe
gray = #6b6b6b
dark-gray = #222222
brown = #7f6d57
blue = #2679a5
green = #19ae38
red= #b92e2e
alert = #b92e2e
volume-min = #a3be8c
volume-med = #ebcb8b
volume-max = #b92e2e
disabled = #6b6b6b
; =======================

[bar/bottom-dk-monitor2]
monitor = ${env:MONITOR:DVI-D-0}
width = 100%
height = 34
radius = 0.0
fixed-center = true
bottom = true
override-redirect = false
;wm-restack = dk

background = ${colors.background}
foreground = ${colors.foreground}

line-size = 2
line-color = #f00

padding-left = 5
padding-right = 5

module-margin-left = 2
module-margin-right = 2

enable-ipc = true

font-0 = "NotoSans Nerd Font:size=11;4"
font-1 = "FiraCode Nerd Font:size=12;4"
font-2 = "FiraCode Nerd Font:size=13;4"
font-3 = "FiraCode Nerd Font:size=14;4"

modules-left   = timeshift-1 timeshift-2 yt-downloads
modules-center = dkw
modules-right  = check-dt-updates check-serv-updates
```

### Polybar log

```text
No issues in logs
```

### Expected behavior

Polybar should be below windows when `override-redirect = false`

### Actual behavior

Loading a window on a particular monitor with polybar's positioned at the top and bottom, The widows appear under the bottom bar and can't see the bottom portion of the window where the bar is positioned.

### Window Manager and Version

dk 2.2

### Linux Distribution

Void Linux

### Polybar version

```text
polybar 3.7.1

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: None
Compiler: /usr/lib/ccache/bin/g++
Compiler flags: -DNDEBUG -fstack-clash-protection -D_FORTIFY_SOURCE=2 -mtune=generic -O2   -g -ffile-prefix-map=/builddir/polybar-3.7.1/build=.  -Wall -Wextra -Wpedantic -Wdeprecated-copy-dtor -Wsuggest-override
Linker flags: -Wl,-z,relro -Wl,-z,now -Wl,--as-needed -Wall -Wextra -Wpedantic -Wdeprecated-copy-dtor -Wsuggest-override  -Wall -Wextra -Wpedantic -Wdeprecated-copy-dtor -Wsuggest-override
```

### Additional Context / Screenshots

I am a normal user of polybar with i3wm and wanted to try something else, Everything works perfect with polybar and i3wm, So I am between this being a dk issue or polybar, Since there are no logs visible on either it is tough to diagnose, I apologize if it is of the wm

**Comment by ACR-Jeff** [comment]
I have tested this issue for some time, It appears dk's config file doesn't play well with including bash scripts, I reconfigured autostart script for dk wm and my polybar launcher script from bash to shell, Now I can get the windows to appear normally after a restart of my system, Load a window and the window still isn't above polybar until after I refresh dk's config file. Then it works as it should. If there is a way to restack on dk wm to prevent this with polybar?
